modules/web-console/frontend/app/services/Messages.service.js

```diff
@@ -42,7 +42,7 @@ export default function factory($alert, errorParser) {
 
         const title = err ? errorMessage(message, err) : errorMessage(null, message);
 
-        msgModal = $alert({type, title, duration});
+        msgModal = $alert({type, title, duration: duration + ((title.match(/<li>/g) || []).length * 3)});
```

Comment:
Lets think about better solution. I do not like regex parsing.

Reply:
Implemented returning of metadata information of error parsing.